controllers/modals/editorDecision/form/EditorDecisionWithEmailForm.inc.php

```diff
@@ -217,7 +226,25 @@ function _sendReviewMailToAuthor($submission, $emailKey, $request) {
 
 		if(is_a($reviewRound, 'ReviewRound')) {
 			// Retrieve review indexes.
-			$reviewAssignmentDao = DAORegistry::getDAO('ReviewAssignmentDAO'); /* @var $reviewAssignmentDao ReviewAssignmentDAO */
+			$reviewAssignmentDao = DAORegistry::getDAO('ReviewAssignmentDAO'); /** @var ReviewAssignmentDAO $reviewAssignmentDao */
+
+			$reviewAssignments = $reviewAssignmentDao->getBySubmissionId($submission->getId(), $reviewRound->getId());
+			$reviewers = [];
+			/** @var ReviewAssignment $reviewAssignment */
+			foreach ($reviewAssignments as $reviewAssignment) {
+				if (in_array($reviewAssignment->getStatus(), [REVIEW_ASSIGNMENT_STATUS_COMPLETE, REVIEW_ASSIGNMENT_STATUS_RECEIVED, REVIEW_ASSIGNMENT_STATUS_THANKED])) {
+					$reviewers[$reviewAssignment->getReviewerId()] = 0;
```

Comment:
Couldn't this just be `$reviewers[] = $reviewAssignment->getReviewerId();` and remove the call to `array_keys` below?

Reply:
Here I wasn't sure if it's possible to have duplicated reviewers... So I was just protecting against the case `array_intersect([1,1], [1,1])`. But as the probability of it happening is near zero, better to simplify 😁

Reply:
It's currently not possible to assign the same reviewer to the same round more than once -- but I do note that it appears to be a possibility in the database, so I've added a foreign key constraint to enforce: https://github.com/pkp/pkp-lib/commit/00907578fa6f18e31a941de25e5842811fc0a471#diff-699762a839da9e614e7f31431ea372d32e07d13690672e824a05015701272418R34